frontend/packages/git-service/src/services/bitbucket-service.ts

```diff
@@ -81,7 +88,7 @@ export class BitbucketService extends BaseService {
   };
 
   isDockerfilePresent = async (): Promise<boolean> => {
-    const url = `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/Dockerfile`;
+    const url = `${this.baseURL}/repositories/${this.metadata.owner}/${this.metadata.repoName}/src/${this.metadata.defaultBranch}/${this.metadata.contextDir}/Dockerfile`;
```

Comment:
This would break if there is no contextDir specified as the URL would then contain `//dockerfile` at the end.

Reply:
This works because the API treats the extra `/` as repo root. I've also updated the relevant nock test calls.